ci-operator/config/openshift/cluster-network-operator/openshift-cluster-network-operator-master.yaml

```diff
@@ -56,6 +56,7 @@ tests:
     cluster_profile: gcp
     upgrade: true
 - as: e2e-ovn-step-registry
+  optional: true
```

Comment:
You want this in `ci-operator/jobs/openshift/cluster-network-operator/openshift-cluster-network-operator-master-presubmits.yaml `

Reply:
@stevekuznetsov I believe I put it in the right place